src/components/Cart/Steps/Steps.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="np-cart__steps__container">
+    <div v-for="(element,index) in steps" :key="index" class="np-cart__step">
+      <div v-if="stepValue - 1 === index" class="np-cart__step__circle--active"/>
```

Comment:
te 3 `divy` mozna zbic do jednego i skorzystac z dynamicznych klas
https://v3.vuejs.org/guide/class-and-style.html#binding-html-classes
to samo z `p`

warunki sie powtarzaja wiec, mozna je wrzucic do metod: `foo(index) { return stepValue - 1 < index }`
jezeli korzystanie z vue-composition-api to bedzie to w `setup`
jutro bede w firmie, moge z tym pomoc